setup.py

```diff
@@ -75,7 +75,8 @@ def check_provided(distribution, min_version, max_version=None, optional=False):
     url="https://github.com/BD2KGenomics/toil-scripts",
     install_requires=[
         'tqdm==3.8.0',  # FIXME: Remove once ADAM stops using it (superfluous import)
-        'pyyaml==3.11'],
+        'pyyaml==3.11',
+        'synapseclient==1.5.1'],
```

Comment:
It's required if we want synapse support. Otherwise, we will get an ImportError.